src/site/xdoc/userguide.xml

```diff
@@ -0,0 +1,214 @@
+<?xml version="1.0" encoding="ISO-8859-1"?>
+<!-- Licensed to the Apache Software Foundation (ASF) under one or more contributor 
+  license agreements. See the NOTICE file distributed with this work for additional 
+  information regarding copyright ownership. The ASF licenses this file to 
+  You under the Apache License, Version 2.0 (the "License"); you may not use 
+  this file except in compliance with the License. You may obtain a copy of 
+  the License at http://www.apache.org/licenses/LICENSE-2.0 Unless required 
+  by applicable law or agreed to in writing, software distributed under the 
+  License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS 
+  OF ANY KIND, either express or implied. See the License for the specific 
+  language governing permissions and limitations under the License. -->
+<document>
+  <properties>
+    <title>Commons Crypto - User guide</title>
+    <author email="dev@commons.apache.org">Commons Documentation Team</author>
+  </properties>
+  <body>
+    <!-- ================================================== -->
+    <section name="User guide">
+      <p>Apache Commons Crypto is a cryptographic library optimized with AES-NI
+        (Advanced Encryption
+        Standard New Instructions). It provides Java API for both cipher level and Java stream
+        level. Developers can use it to implement high performance AES encryption/decryption with
+        the minimum code and effort. Please note that Apache Commons Crypto doesn't implement the cryptographic
+        algorithm such as AES directly. It wraps to Openssl or JCE which implement the algorithms.
+      </p>
+      <subsection name="Interfaces Overview">
+        <p>Interfaces and classes used by the various implementation in the sub-packages.</p>
+        <table>
+          <tr>
+            <td width="150">
+              <a href="apidocs/index.html">cipher
+              </a>
+            </td>
+            <td>
+              The interface of cryptographic cipher for encryption and decryption.
+            </td>
+          </tr>
+
+          <tr>
+            <td width="150">
+              <a href="apidocs/index.html">random
+              </a>
+            </td>
+            <td>
+              The interface for SecureRandom.
+            </td>
+          </tr>
+
+          <tr>
+            <td width="150">
+              <a href="apidocs/index.html">stream
+              </a>
+            </td>
+            <td>
+              The interface wraps the underlying stream and it automatically encrypts the stream when data is written and decrypts the stream when data is read.
+            </td>
+          </tr>
+
+        </table>
+      </subsection>
+      <subsection name ="Usage">
+        <ol style="list-style-type: decimal">
+        <h4>Usage of Random API</h4>
+        <table>
```

Comment:
Better to add the introduction：  SecureRandom provides a cryptographically strong random number generator. The default implementation will use `Intel® Digital Random Number Generator (DRNG)` for accelerate the random generation